rpcs3/Emu/Cell/SPUInterpreter.cpp

```diff
@@ -1899,18 +1902,19 @@ static void DFASM(SPUThread& spu, spu_opcode_t op, DoubleOp operation)
 			case DFASM_S: result = a - b; break;
 			case DFASM_M: result = a * b; break;
 			}
-			if (fetestexcept(FE_INVALID))
+			const u32 e = _mm_getcsr();
+			if (e & FE_INVALID)
 			{
 				spu.fpscr.setDoublePrecisionExceptionFlags(i, FPSCR_DINV);
 				result = DOUBLE_NAN;
 			}
 			else
 			{
-				if (fetestexcept(FE_OVERFLOW))
+				if (e & FE_OVERFLOW)
```

Comment:
_mm_getcsr is not compatible with these constants.
There are _MM_MASK_OVERFLOW and similar ones.